sample/build.gradle

```diff
@@ -11,6 +11,7 @@ android {
         targetSdkVersion 28
 
         proguardFiles getDefaultProguardFile('proguard-android.txt')
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
```

Comment:
Couldn't figure out a way to minify Joda itself in the `library` module - minification would only affect `library` module code - so added tests for this in `sample` instead.

Reply:
Hah, seems reasonable enough to me.